reference/exercise-concepts/bank-account.md

```diff
@@ -114,7 +114,7 @@ end
 
 ### `send/2` and `receive/1`
 
-[by @opsb (adapted)](https://exercism.io/tracks/elixir/exercises/bank-account/solutions/d8d4ab8d4b2b47509cf1976fb93da0a1)
+[by @opsb (adapted)](https://exercism.org/tracks/elixir/exercises/bank-account/solutions/opsb)
```

Comment:
Will it work when they make a profile?

Reply:
Yes, except that maybe I got the source of the error wrong. It looks like it's this one: https://github.com/exercism/exercism/issues/5636

The solution has comments, and I cannot open any community solutions with comments either... I'm positive this will start working